common/src/main/java/com/datastrato/gravitino/config/ConfigEntry.java

```diff
@@ -134,6 +140,20 @@ public ConfigEntry<Optional<T>> createWithOptional() {
     return conf;
   }
 
+  /**
+   * Creates a new ConfigEntry instance based on this configuration entry with no default value.
+   *
+   * @return A new ConfigEntry instance with no default value.
+   */
+  public ConfigEntry<T> createWithNoDefault() {
```

Comment:
I think `create` is enough, no need to specify with "WithNoDefault".

Reply:
OK, will fix.